### PR TITLE
Problem: issue #942 addressed with feature to add zoom reset

### PR DESCRIPTION
**Pull request by alexandercbooth**
Solution: Added zoom reset to the menu and mapped it to the suggested keys, CmdOrCtrl+0.

### CODE REVIEW

File: src/main/menu.js [hunk 1]
```diff
@@ -273,6 +273,11 @@ export const view = {
       click: createSender('menu:zoom-out'),
     },
     {
+      label: 'Zoom Reset',
```
Comment (rgbkrk) [inline src/main/menu.js]: To match the Chrome menu, I'd make this "Actual Size"

screen shot 2016-10-08 at 11 14 31 am

(I didn't notice this until a second ago)

Comment (alexandercbooth) [reply src/main/menu.js]: Good thinking, I will change it right away. Would you mind walking me through the best way to do this commit-wise (should I just do another commit and then rebase?).

Thanks!

Comment (rgbkrk) [reply src/main/menu.js]: New commits are fine, we don't need things squashed here.

File: src/notebook/menu.js [hunk 2]
```diff
@@ -225,6 +225,10 @@ export function dispatchZoomOut() {
   webFrame.setZoomLevel(webFrame.getZoomLevel() - 1);
 }
 
+export function dispatchZoomReset() {
+  webFrame.setZoomLevel(1);
+}
```
Comment (rgbkrk) [inline src/notebook/menu.js:230]: Would you want to tackle the test for this? There should be a similar test for dispatchZoomOut you can borrow from.

Comment (alexandercbooth) [reply src/notebook/menu.js:230]: Absolutely, happy to!

Comment (rgbkrk) [reply src/notebook/menu.js:230]: By the way, feel free to find me in the table area in the Atrium at PyData DC.

File: test/renderer/menu-spec.js [hunk 3]
```diff
@@ -118,6 +118,15 @@ describe('menu', () => {
     });
   });
 
+  describe('dispatchZoomReset', () => {
+    it('executes zoom reset', () => {
+      const setZoomLevel = sinon.spy(webFrame, 'setZoomLevel');
+      menu.dispatchZoomReset();
+      setZoomLevel.restore();
+      expect(setZoomLevel).to.be.called;
```
Comment (rgbkrk) [inline test/renderer/menu-spec.js:126]: Awesome thing we can do here:

``` js
expect(setZoomLevel).to.be.calledWith(1);
```

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Super awesome, thank you.

File: test/renderer/menu-spec.js (hunk 3 above)
Comment (rgbkrk) [reply test/renderer/menu-spec.js:126]: I just merged this, but you can make another PR if you like.

File: src/notebook/menu.js (hunk 2 above)
Comment (alexandercbooth) [reply src/notebook/menu.js:230]: Will do!

**Comment by alexandercbooth** [comment]
My pleasure, thanks for the help!

File: test/renderer/menu-spec.js (hunk 3 above)
Comment (alexandercbooth) [reply test/renderer/menu-spec.js:126]: Happy to, what would you suggest for commit message, chore, test, refactor (would have asked in person but had to leave early today, sorry)?

**Pull request merged**